### PR TITLE
Feature/minor style fixes

### PR DESCRIPTION
**Pull request by MatthewScholefield**
This PR has some small style fixes.

### CODE REVIEW

File: msm/skill_entry.py [hunk 1]
```diff
@@ -217,9 +218,8 @@ def run_pip(self, constraints=None):
 
         LOG.info('Installing requirements.txt for ' + self.name)
         can_pip = os.access(dirname(sys.executable), os.W_OK | os.X_OK)
-        pip_args = [
-            sys.executable, '-m', 'pip', 'install'
-        ] + self.dependent_python_packages
+        pip_args = [sys.executable, '-m', 'pip', 'install']
+        pip_args += [self.dependent_python_packages]
```
Comment (forslund) [inline msm/skill_entry.py]: I think this causes an issue. `self.dependent_python_packages` is a list so this adds the entire list as a single argument to Popen.

**Review by forslund — CHANGES_REQUESTED** [review]
Nice improvement of code readability. Found an issue with the pip install, apart from that nothing to complain on.

File: msm/skill_entry.py (hunk 1 above)
Comment (MatthewScholefield) [reply msm/skill_entry.py]: Wow, I'm stupid! Nice catch! Updated.

**Comment by forslund** [comment]
Looks good now. Merging

**Pull request merged**